### ISSUE TITLE
Missing alpha values for points in point cloud data

### ISSUE THREAD

**Issue by JiachengDeng** [buildingnet/buildingnet_dataset#12]
When I follow the steps given by the author to pre-process the data, 
`python -m lib.datasets.preprocessing.buildingnet`
the following error occurs:
```
File "/ssd/djc/Challenge/buildingnet_dataset/MinkowskiNet/lib/datasets/preprocessing/buildingnet.py", line 50, in <module>
    save_point_cloud_with_normals(processed, out_filepath, with_label=True, verbose=False)
  File "/ssd/djc/Challenge/buildingnet_dataset/MinkowskiNet/lib/pc_utils.py", line 79, in save_point_cloud_with_normals
    assert points_3d.shape[1] == 11
AssertionError
```
I checked through debug mode and found that the BuildingNetV1 dataset seems to be missing alpha values, resulting in only 10 rather than 11 dimensions of Coordinates(3)/Normals(3)/RGB(3)/Label(1) information for each point.
Looking forward to the author's reply!

**Comment by buildingnet** [comment]
The alpha values were removed from the BuildingNet v1 dataset, since in most cases points had the same value (```alpha=1.0```). The current MinkowskiNet implementation was build based on the v0 dataset. You will need to make some changes to be able to train the network using the current code e.g. change [lines 78-82](https://github.com/buildingnet/buildingnet_dataset/blob/6e97ca48f584530fc325c8d0708456b1f0870767/MinkowskiNet/lib/pc_utils.py#L78-L81) in the ```MinkowskiNet/lib/pc_utils.py``` to this:
```python
assert points_3d.shape[1] == 10
python_types = (float, float, float, float, float, float, float, float, float, int)
npy_types = [('x', 'f4'), ('y', 'f4'), ('z', 'f4'), ('nx', 'f4'), ('ny', 'f4'), ('nz', 'f4'), ('red', 'f4'),
             ('green', 'f4'), ('blue', 'f4'), ('label', 'u1')]
````

We are currently working on releasing the MinkNet implementation for the v1 dataset, were we also upgraded the Minkowski Engine to the latest version (v0.5.4).
